### ISSUE TITLE
4L 회고 / 23.06.29 / Day17 🦾✍💪

### ISSUE THREAD

**Issue by aaingyunii** [aaingyunii/Bootcamp_TIL#17]
## Liked
-  머신 러닝의 개요를 들으면서 예전에 학습 했던 내용들이 조금씩 떠올라서 좋았습니다. 
- 데이터 시각화 수업을 들으면서 언급됐던 데이터 분석의 본격적인 학습을 할 수 있다는 점이 좋았습니다.

## Lacked
- 초반에 이론 설명 부분이 많아 피로감을 느껴서 아쉬웠습니다.

## Learned
- 머신러닝과 딥러닝의 차이를 조금은 알 수 있었습니다.
- 데이터 관련 용어를 배웠습니다.
```
### Feature (X)
- 추론하기 위한 근거가 되는 값들을 표현하는 용어.
- 예측 하거나 분류해야 하는 데이터의 특성, 속성 값을 말한다.
- 입력 변수(Input), 독립변수라고도 한다.
- 일반적으로 X로 표현한다.

### Label (y)
- 예측하거나 분류해야 하는 값들을 표현하는 용어
- 출력 변수(Output), 종속변수, Target 이라고도 한다.
- 일반적으로 y로 표현한다.

### Class
- Label이 범주형(Categorical) 일 경우 그 고유값을 Class라고 한다.
    - 개/고양이, 남성/여성, A형/AB형/B형/O형

### 데이터 포인트
- 개별 데이터를 표현하는 용어. 
```
<br>

- 이전에 `Numpy` 모듈의 `random.seed()` 의 개념을 알 수 있었고, `train_test_split()` 메서드의 파라미터인 `random_state`가 존재하는 이유, 설정하는 의미가 무엇인지 학습할 수 있었습니다.

```
#### train_test_split에서 random_state(random seed) 설정의 의미 :

전체 데이터에서 train과 test 데이터로 무작위로 나누고 여러 모델에 학습하여 테스트할 때,
무작위로 데이터를 뽑아서 각기 다른 데이터 셋으로 모델들을 테스트하는 것은 각각의 모델들의 신뢰성과 연관이 없어지기 때문에
무작위로 뽑는 데이터 셋들을 random_state를 통해 모두가 일정하게 무작위로 뽑힌 데이터 셋을 사용하게 함으로써 
예측 모델들의 평가의 신뢰성을 주기 위함이다.
```

## Longed for
- 오늘 배운 내용들을 정리해서 블로그에 업로드 할 예정입니다.
- Github에 수업에 실행한 예제 코드를 다시 실행하여 따로 파일을 만들어 업로드할 것입니다.
